### PR TITLE
CartoDB::Visualization::Member out of everywhere but old models [WIP]

### LINKED ISSUE #12844: [Model refactor] More Visualization::Member removal
Remove Visualization::Member removal everywhere outside old models, and Visualization::Collection if possible.

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #12844.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
retest this please

**Comment by juanignaciosl** [comment]
Check my last commit, it was an unused variable

File: app/services/carto/visualizations_export_persistence_service.rb [hunk 1]
```diff
@@ -106,9 +106,7 @@ def save_import(user, visualization, renamed_tables: {}, full_restore: false)
         end
       end
 
-      # Propagate changes (named maps, default permissions and so on)
-      visualization_member = CartoDB::Visualization::Member.new(id: visualization.id).fetch
-      visualization_member.store
+      visualization.save!
```
Comment (javitonino) [inline app/services/carto/visualizations_export_persistence_service.rb]: FYI @amartinj (you have a similar change in your user migrator PR). Keep an eye out for conflicts.

**Comment by javitonino** [comment]
I did a classical quick-smokes of visualizations management and overlays, and seems to work. 🚀

**Pull request merged**